opensplice_cmake_module/cmake/Modules/FindOpenSplice.cmake

```diff
@@ -76,7 +79,7 @@ if(NOT _ospl_home STREQUAL "")
     "ddsserialization"
     "ddsuser"
     "ddsutil"
-    "durability"
+    # "durability"
```

Comment:
Good point. As far as I know it doesn't exist anymore. I will update the commit which commented it out to actually remove the line.